src/ProjectTracker.ts

```diff
@@ -0,0 +1,50 @@
+import * as _path from 'path';
+import * as fs from 'fs';
+
+export class ProjectTracker {
```

Comment:
is there a need to export this if we're intending it to be used as a singleton? as far as I can tell from purduesigbots/pros-atom3@a84fe9ae93886f58f5b1b9026c853423ee48b284 we should only need the instance

Reply:
If an editor wants to have their own ProjectTracker for some reason? Perhaps they're running one instance of Node that's shared across multiple windows? Not sure I have a concrete reason now.

Reply:
in that case, I think we should leave it unexported until such time as there is a need to expose it

Reply:
```suggestion
class ProjectTracker {
```